libraries/GCS_MAVLink/GCS.h

```diff
@@ -610,7 +610,14 @@ class GCS_MAVLINK
      */
     uint32_t correct_offboard_timestamp_usec_to_ms(uint64_t offboard_usec, uint16_t payload_size);
 
-    static void convert_COMMAND_LONG_to_COMMAND_INT(const mavlink_command_long_t &in, mavlink_command_int_t &out);
+    // converts a COMMAND_LONG packet to a COMMAND_INT packet, where
+    // the command-long packet is assumed to be in the supplied frame.
+    // If location is not present in the command then just omit frame.
+    static void convert_COMMAND_LONG_to_COMMAND_INT(const mavlink_command_long_t &in, mavlink_command_int_t &out, MAV_FRAME frame = MAV_FRAME_GLOBAL_RELATIVE_ALT);
+
+    // methods to extract a Location object from a command_long or command_int
+    bool location_from_command_t(Location &out, const mavlink_command_long_t &in, MAV_FRAME frame);
```

Comment:
any reason why the in and out orders are different between these two commands?  convert_COMMAND_LONG_to_COMMAND_INT vs location_from_command_t

Reply:
I've reordered and renamed the parameters as requested.